stream/cards.md

```diff
@@ -16,7 +16,30 @@ To facilitate users linking multiple cards, developers can add identifying key-v
 
 ### What Is Card Verification
 
-When linking a card, cardholders are required to complete a verification process to verify card ownership to ensure we protect them. The verification consists of the following steps:
+When linking a card, cardholders are required to complete one of the two available verification processes to verify card ownership to ensure we protect them.
+
+When linking a card, a property `verificationMethod` will identify what verification method that card is meant to follow - `3ds` or `microcharge`.
```

Comment:
does it make sense to explain that this property is returned from the create card endpoint?

Reply:
yeah, I'll add a note